File: mo-system/mo-system-api/src/main/java/ink/ckx/mo/system/api/model/query/RolePageQuery.java
package ink.ckx.mo.system.api.model.query;

import ink.ckx.mo.common.core.base.BasePageQuery;
import io.swagger.v3.oas.annotations.media.Schema;
import lombok.Data;

/**
 * 角色分页Query
 *
 * @author chenkaixin
 */
@Data
public class RolePageQuery extends BasePageQuery {

    @Schema(description = "关键字(角色名称/角色编码)")
    private String keywords;
}

File: mo-admin/mo-admin-api/src/main/kotlin/ink/ckx/mo/admin/api/model/query/RolePageQuery.kt
package ink.ckx.mo.admin.api.model.query

import ink.ckx.mo.common.core.base.BasePageQuery
import io.swagger.v3.oas.annotations.media.Schema

/**
 * 角色分页Query
 *
 * @author chenkaixin
 */
@Schema(description = "角色分页Query")
data class RolePageQuery(

    @Schema(description = "关键字(角色名称)")
    var keywords: String? = null,
) : BasePageQuery()